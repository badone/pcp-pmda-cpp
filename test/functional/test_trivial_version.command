./pmdatrivial --version | sed -re 's/^((PCP|PMDA interface) version ).*$/\1<redacted>/'
