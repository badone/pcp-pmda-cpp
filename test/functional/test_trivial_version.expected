
trivial PMDA
PMDA interface version 6

